contracts/staking/UniswapV2SwapHelper.sol

```diff
@@ -6,10 +6,6 @@ import "../Interfaces.sol";
 import "./SimpleStaking.sol";
 
 library UniswapV2SwapHelper {
-	function getRouter(SimpleStaking staking) public view returns (Uniswap) {
-		return Uniswap(staking.nameService().getAddress("UNISWAP_ROUTER"));
-	}
-
```

Comment:
revert all changes in this file except for the usage of Interface instead of SimpleStaking